packages/liquid-html-parser/grammar/liquid-html.ohm

```diff
@@ -127,8 +127,13 @@ Liquid <: Helpers {
   liquidTagLiquidMarkup = tagMarkup
 
   liquidTagContentFor = liquidTagRule<"content_for", liquidTagContentForMarkup>
+  
   liquidTagContentForMarkup =
-    contentForType (argumentSeparatorOptionalComma tagArguments) (space* ",")? space*
+
+    contentForType (argumentSeparatorOptionalComma contentForTagArgument) (space* ",")? space*
+    contentForTagArgument = listOf<contentForNamedArgument<delimTag>, argumentSeparatorOptionalComma>
+    contentForNamedArgument<delim> = (variableSegment "."? variableSegment?) space* ":" space* (liquidExpression<delim>)
```

Comment:
Is this because the `.` is only accepted in content_for named argument parsing?

Reply:
Yes indeed. I don't believe other tag attributes need to support `.` atm. This is why I had to create new contentForTagArgument